CHANGES.rst

```diff
@@ -13,6 +13,8 @@ assign_wcs
 - Updated assign_wcs to compute the sky footprint of MIRI MRS and Nirspec
   IFU observations. [#2472]
 
+- NIRISS models updated to allow for negative filter wheel rotations [#1801]
```

Comment:
This entry should go under `transforms`.

Reply:
:+1: